### DISCUSSION TITLE
端口显示被占用是为什么呢？

### DISCUSSION THREAD

**Discussion by Chiyey** [GaiZhenbiao/ChuanhuChatGPT#535] [Q&A]
![image](https://user-images.githubusercontent.com/124230156/229675671-f9e3b378-d1d9-47ce-b8cc-25d094619aa1.png)
把端口和代理设为一致了已经。

**Comment by maxixi13** [comment]
那你就换个端口呀

**Reply by Chiyey** [reply]
换了也一样的呀

**Reply by Chiyey** [reply]
再往深我就不会了
最开始是显示目标计算机积极拒绝

**Reply by Chiyey** [reply]
![image](https://user-images.githubusercontent.com/124230156/229678174-03e5144a-acbc-434f-b61c-f1c9bf4b0c4e.png)
这是更换了端口的截图

**Reply by maxixi13** [reply]
来个联系方式 我教你

**Reply by maxixi13** [reply]
虽然我的 还没成功 哈哈哈哈哈

**Reply by Chiyey** [reply]
哈哈哈哈，我才刚刚看到你的信息你现在弄好了吗

**Comment by Keldos-Li** [comment]
![image](https://user-images.githubusercontent.com/23137268/230115615-7475db01-359d-4f98-9536-492bcdddefd0.png)
不直接设置系统代理的，复制config-example.json到自己的config.json时把proxy相关的这两行删掉，example中的值只是示例

**Reply by Chiyey** [reply]
<img width="857" alt="image" src="https://user-images.githubusercontent.com/124230156/230274120-bec37874-5a5b-4224-87ad-3c5362f368c4.png">
我更改了但是现在是这样

**Reply by Chiyey** [reply]
哎哎哎
现在情况有变 更改了但进去直接报错
<img width="1275" alt="image" src="https://user-images.githubusercontent.com/124230156/230275120-4a6fecec-032a-46c3-b5be-852d0f3b8c96.png">
